source/includes/Overview/_authentication.md.erb

```diff
@@ -34,7 +34,7 @@ Once the user is directed to access the authorization endpoint at TSheets ('auth
 
  The link that a user will follow to perform an authorization request is made to the /authorize end-point:
 
- <%= partial "includes/Overview/Examples/authorize.tmpl.erb" %>
+<%= partial "includes/Overview/Examples/authorize.tmpl.erb" %>
```

Comment:
FYI, not sure why, but the leading space was causing a stray "</span>" tag to be rendered literally.  Something in Slate's processing.  Removing the space fixes it.